state/cassandra/cassandra.go

```diff
@@ -240,7 +242,7 @@ func (c *Cassandra) Get(ctx context.Context, req *state.GetRequest) (*state.GetR
 		session = sess
 	}
 
-	results, err := session.Query(fmt.Sprintf("SELECT value FROM %s WHERE key = ?", c.table), req.Key).WithContext(ctx).Iter().SliceMap()
+	results, err := session.Query(fmt.Sprintf("SELECT value, TTL(value) AS ttl, toTimestamp(now()) AS now FROM %s WHERE key = ?", c.table), req.Key).WithContext(ctx).Iter().SliceMap()
```

Comment:
Yeah this is what I was actually intending to do.. but couldn't figure out the Cassandra syntax either. I'll have another crack at it but this might be what we have to do.

Yes, the TTL command is dynamic https://docs.datastax.com/en/cql-oss/3.3/cql/cql_using/useTTL.html